index.html

```diff
@@ -1,69 +1,269 @@
+                  <li class="work__item">2015</li>
+                </ul>
+              </div>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="portfolio__tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+              </ul>
+              <div>
+                <a class="button_primary" href="#">See project</a>
+              </div>
+            </div>
+          </article>
+          <article class="work_card">
+            <header class="work__header">
+              <img
+                src="assets/snapshoot3.png"
+                alt="Snapshoot Portfolio 3"
+                class="work__image"
+              />
+            </header>
+            <div class="portfolio__content">
+              <h3 class="portfolio__title color-dark">Tonic</h3>
+              <div class="portfolio__profile">
+                <h4 class="portfolio__author color-info">Canopy</h4>
+                <ul class="portfolio__list">
+                  <li class="work__item">Back End Dev</li>
+                  <li class="work__item">2015</li>
+                </ul>
+              </div>
+              <p class="portfolio__description">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="portfolio__tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+              </ul>
+              <div>
+                <a class="button_primary" href="#">See project</a>
+              </div>
+            </div>
+          </article>
+          <article class="work_card">
+            <header class="work__header">
+              <img
+                src="assets/snapshoot4.png"
+                alt="Snapshoot Portfolio 4"
+                class="work__image"
+              />
+            </header>
+            <div class="portfolio__content">
+              <h3 class="portfolio__title color-dark">Multi-Post Stories</h3>
+              <div class="portfolio__profile">
+                <h4 class="portfolio__author color-info">Canopy</h4>
+                <ul class="portfolio__list">
+                  <li class="work__item">Back End Dev</li>
+                  <li class="work__item">2015</li>
+                </ul>
+              </div>
+              <p class="portfolio__description">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="portfolio__tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+              </ul>
```

Comment:
- [ ] You made progressive efforts which I must commend 👏 . However, I suggest you make a few adjustments to this section of the cards in reference to the Figma design. 
### Points to note:
- The `h4` font is `Poppins`. kindly adjust it to fit the design given
- Kindly align the `ul` section that has languages to move to the left and align with the card elements

- [ ] Please consider revising your `flexbox` materials for the alignment 👍 __

### Figma:
![image](https://github.com/montsecruz9/personal-portfolio/assets/50990925/ecb16692-ea89-4c7f-92f5-54f11dbcf0e1)

### Your work:
![image](https://github.com/montsecruz9/personal-portfolio/assets/50990925/ecb61d37-de0b-4318-9230-efcea1763654)

Reply:
> * [ ]  You made progressive efforts which I must commend 👏 . However, I suggest you make a few adjustments to this section of the cards in reference to the Figma design.
> 
> ### Points to note:
> * The `h4` font is `Poppins`. kindly adjust it to fit the design given
> * Kindly align the `ul` section that has languages to move to the left and align with the card elements
> * [ ]  Please consider revising your `flexbox` materials for the alignment 👍 __
> 
> ### Figma:
> ![image](https://user-images.githubusercontent.com/50990925/244177706-ecb16692-ea89-4c7f-92f5-54f11dbcf0e1.png)
> 
> ### Your work:
> ![image](https://user-images.githubusercontent.com/50990925/244177922-ecb61d37-de0b-4318-9230-efcea1763654.png)

- [ ] For the above comment, kindly fix the space after `CANOPY`, it seems the spaces between the elements are equal kindly adjust it.

Here is your current design.

<img width="292" alt="image" src="https://github.com/montsecruz9/personal-portfolio/assets/41163571/fc7485b1-5611-44ee-bb44-760fc0c4c3e0">